[DOCS] Adds feature importance to regression and classification docs (#844)
elastic/stack-docs: docs/en/stack/ml/shared-ml-concepts.asciidoc

tag::feature-importance[]

> Feature importance is calculated for supervised {ml} processes such as 

Reviewer: ```suggestion
Feature importance is calculated for supervised {ml} methods such as 
```

{regression} and {classification}. This value provides further insight into the 

> results of a {dfanalytics} and therefore helps interpret these results. As we 

Reviewer 2: The presence of "a" in this sentence makes me think it should be referring to "a job". If not, the "a" should be removed:
```suggestion
results of a {dfanalytics-job} and therefore helps interpret these results. As we 
```

mentioned, there are multiple features of a data point that are analyzed during 
{dfanalytics}. These features are responsible for a particular prediction to 

> varying degrees. Feature importance shows on what degree a given feature of a 

Reviewer 2: ```suggestion
varying degrees. Feature importance shows to what degree a given feature of a 
```

datapoint contributes to the prediction. The feature importance value of a 
feature can be either positive or negative depending on its effect on the 

> prediction; the value is negative if the feature reduces the prediction value 

Reviewer 2: Not mandatory, but I'd recommend splitting this:
```suggestion
prediction. If the feature reduces the prediction value, the feature importance is negative.
```

> and positive if the feature increases the prediction value. The magnitude of the 

Reviewer 2: Ditto:
```suggestion
If the feature increases the prediction value, the feature importance is positive. The magnitude of the 
```

> feature importance value shows how significant is the effect of the given 

Reviewer 2: ```suggestion
feature importance value shows how significantly the
```

> feature on the prediction both locally (for a given datapoint) or generally (for 

Reviewer 2: ```suggestion
feature affects the prediction both locally (for a datapoint) or generally (for 
```
